test/e2e/serial/workload_placement_test.go

```diff
@@ -1129,3 +1278,20 @@ func labelNode(cli client.Client, label, nodeName string) (func() error, error)
 
 	return unlabel, nil
 }
+
+func maxResourceType(nrtInfo nrtv1alpha1.NodeResourceTopology, resName corev1.ResourceName) resource.Quantity {
+
+	var max resource.Quantity
+
+	for _, zone := range nrtInfo.Zones {
+		zoneQty, ok := e2enrt.FindResourceAvailableByName(zone.Resources, resName.String())
+		if !ok {
+			continue
+		}
+
+		if zoneQty.Cmp(max) > 1 {
+			max = zoneQty.DeepCopy()
```

Comment:
or just assign, which should be sufficient to avoid side effects (see comment below) and do `DeepCopy` just one time at return time